.travis.yml

```diff
@@ -23,8 +23,10 @@ env:
   matrix:
   - LINT_CHECK="1"
   - TRANSIFEX="1"
-  - TESTS="1" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
-  - TESTS="1" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
+  - TESTS="1" ODOO_REPO="odoo/odoo" LINT_CHECK="0" EXCLUDE=sale_sourced_by_line,sale_procurement_group_by_line
+  - TESTS="1" ODOO_REPO="OCA/OCB" LINT_CHECK="0" EXCLUDE=sale_sourced_by_line,sale_procurement_group_by_line
```

Comment:
Why excluding those modules?

Reply:
Hi @mreficent,

as I wrote [here](https://github.com/tafaRU/sale-workflow/tree/10.0-mig-sale_mrp_link/sale_mrp_link#id1), this module is incompatible with those modules. The reason for this is that `sale.order` and `mrp.production` are linked by the field `sale_order_id` and when the module `sale_procurement_group_by_line` is installed, the field `procurement_group_id` is only filled at `sale.order.line` [1] level not at `sale.order` level, as a standard way [2], and, consequently, [here](https://github.com/tafaRU/sale-workflow/blob/10.0-mig-sale_mrp_link/sale_mrp_link/models/procurement_order.py#L16-L17) no `sale.order` is found.

After https://github.com/OCA/sale-workflow/pull/573 the `sale_procurement_group_by_line` module [is installed anyway](https://travis-ci.org/OCA/sale-workflow/jobs/397259086#L1790), even though I exluded it. So that, I'd also to `EXCLUDE` the module `sale_stock_picking_blocking_proc_group_by_line`.

Would that be fine by you?

[1] https://github.com/OCA/sale-workflow/blob/10.0/sale_procurement_group_by_line/model/sale.py#L83
[2] https://github.com/odoo/odoo/blob/10.0/addons/sale/models/sale.py#L669

Reply:
just seen https://github.com/OCA/sale-workflow/pull/690 it also should work in my case!